FilteringApp/FilteringApp.Core/Models/ArrayParseResult.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
```

Comment:
As far as I see, these usings are unused. There is a cool way to do so: on Visual Studio for Windows there are couple of shortcuts:

ctrl + K+ D --> formats the file
ctrl + R + G --> removes unused usings

Reply:
Nice trick, fixed.